cmd/k8s-operator/deploy/chart/values.yaml

```diff
@@ -22,6 +22,16 @@ operatorConfig:
   defaultTags:
     - "tag:k8s-operator"
 
+  # Manual configuration of Tailscaled port
+  port:
+    enabled: false
+    containerPort: 8080
+
+  # Optional host network configuration. Likely only needed for public Kubernetes nodes
+  hostNetwork:
+    enabled: false
+    dnsPolicy: ClusterFirstWithHostNet  # Common options: ClusterFirst, ClusterFirstWithHostNet, Default
```

Comment:
I would prefer not to expose this as configurable unless we have a specific need for it. Can we just set it to `ClusterFirstWithHostNet` if `hostNetwork` is enabled, and otherwise leave it unset, or do you already have a use-case where that default won't work?